earth_enterprise/rpms/opengee-fusion/pre-uninstall.sh

```diff
@@ -21,6 +21,10 @@ set +x
 #-----------------------------------------------------------------
 
 service gefusion stop
```

Comment:
You don't need to stop the service if this is an upgrade. You should only be doing things that don't effect a running service at this point. Or if you're supper worried about that, you can stop it, do your work, and then start it again.